tests/hal/test_cpu.py

```diff
@@ -238,7 +238,7 @@ def test_hal_cpu_get_TSEG_is_server_true_cmd(self):
         reg_call_1 = call('8086.MEMMAP_VTD.TSEG')
         reg_call_2 = call('8086.MEMMAP_VTD.TSEG')
         field_call_1 = call('base', True)
-        field_call_2 = call('limit', True)

```

Comment:
If you confirm that undoing the cpu hal change above, then this will need to be reverted too.